### ISSUE TITLE
Accessibility: Scrollable Regions

### ISSUE THREAD

**Issue by pancakereport** [jupyter-book/myst-theme#743]
Scrollable regions (like wide code cells) are not accessible via keyboard. An example is the cell defining `biggest_difference()` [here](https://inferentialthinking.com/chapters/08/functions-and-tables/) with output from axe accessibility checker [here](https://github.com/data-8/textbook/actions/runs/20148056555/job/57833764033#step:9:62). The checker is configured to meet the standards required by ADA Title II.

I'm more than happy to help with this fix, and am filing this issue for tracking purposes.

The way to fix this is simply to add `tabindex="0"` (or the appropriate number).

**Comment by pancakereport** [comment]
Weirdly running to code output (data frame/table) not having tabindex=0 here: https://data88e.org/textbook/content/introduction/

Selector: div[data-output-id="FFNAhMznel9APS72eCOMs"] > .thebe-output > .jp-OutputArea.lm-Widget > .lm-Panel.jp-OutputArea-child.lm-Widget > .jp-RenderedHTMLCommon.jp-RenderedHTML.jp-mod-trusted

**Comment by pancakereport** [comment]
Another example of table output from a code cell not being scrollable-region-focusable: https://ds100.org/course-notes/pca/
